testutil/ibctesting/coordinator.go

```diff
@@ -35,7 +35,7 @@ func NewCoordinator(t *testing.T, n int) *Coordinator {
 	chains := make(map[string]*TestChain)
 
 	for i := 0; i < n; i++ {
-		chainID := GetChainID(i)
+		chainID := GetChainID(i + 1)
```

Comment:
Why plus 1 here?

Reply:
This is just to reflect the following change that has been made inside the standard `Coordinator` implementation with `ibc-go v3.0.0`: 

> (testing) https://github.com/cosmos/ibc-go/pull/774 Added ChainID arg to SetupWithGenesisValSet on the testing app. Coordinator generated ChainIDs now starts at index 1